backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/ScheduleSpecification.java

```diff
@@ -27,11 +27,8 @@
 @Getter @Accessors(fluent = true)
 public class ScheduleSpecification {
 
-    @NotNull(message = "Inbound route id must not be null")
-    private Long inboundRouteId;
-
-    @NotNull(message = "Outbound route id must not be null")
-    private Long outboundRouteId;
+    @NotNull(message = "Line id must not be null")
+    private Long lineId;
```

Comment:
See BackendIdHandlerMethodArgumentResolver#resolveArgument how to extract id from URL. Especially ```UriUtils.findMappingVariable("id", parameter.getMethod(), lookupPath) ``` part